contest/contest_info.py

```diff
@@ -43,7 +45,7 @@ def get_total_testcases(problem):
     testcases = Testcase.objects.filter(problem = problem)
     return testcases.count()
 
-def get_contestant_problem_submission_list(contest,contestant,problem):
+def get_contestant_problem_submission_list(contest, contestant, problem):
     return Submission.objects.filter(problem = problem, submit_time__lte = contest.end_time,
         submit_time__gte = contest.start_time,user = contestant.user).order_by('submit_time')
```

Comment:
Maybe sorted by ``-submit_time`` will make it easier for users to find their latest submissions.